packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3907,6 +3929,11 @@ class SemanticsConfiguration {
   /// configurations as determined by [isCompatibleWith].
   void absorb(SemanticsConfiguration child) {
     assert(!explicitChildNodes);
+    assert(
```

Comment:
Is this condition already ensured by isCompatibleWith? isCompatibleWith should probably mark configurations that have different _editableTextIds as incompatible?

Reply:
Done. I would also like to assert that no two editables are nested. Is there a good place for such an assertion?

Reply:
Although, I suppose a single text field can be described by two nodes each supplying a different subset of semantic properties, so maybe we should allow it.